odoo/tools/view_validation.py

```diff
@@ -18,6 +18,13 @@
 
 READONLY = re.compile(r"\breadonly\b")
 
+# predefined symbols for evaluating attributes (invisible, readonly...)
+# this symbols are deprecated, a warning will be add to the console.
+DEPRECATED_IGNORED_IN_EXPRESSION = {
```

Comment:
Maybe it is possible to put it in a test so as not to have code that is there only to make a deprecation warning?